.prettierrc

```diff
@@ -1,4 +1,11 @@
 {
-  "singleQuote": true
+  "arrowParens": "always",
+  "bracketSpacing": true,
+  "jsxSingleQuote": false,
+  "printWidth": 100,
+  "semi": true,
+  "singleQuote": true,
+  "tabWidth": 2,
+  "useTabs": false,
+  "trailingComma": "none"
```

Comment:
I've found `trailingComma` to be beneficial for reducing git diff (this pr included), do you have a strong opinion on this?

Reply:
I don't have a *strong* opinion. I guess it's just habit, and something I'm more used to.

Reply:
Changed to `"trailingComma": "es5"` (which is the default for prettier).
Re-formatted all files with this option in place.